test/unit/Reflection/ReflectionPropertyTest.php

```diff
@@ -362,4 +367,44 @@ public function testGetStartColumnAndEndColumn(string $php, int $startColumn, in
         self::assertEquals($startColumn, $constantReflection->getStartColumn());
         self::assertEquals($endColumn, $constantReflection->getEndColumn());
     }
+
+    public function testGetAst() : void
+    {
+        $php = '<?php
```

Comment:
Can I suggest using `<<<'PHP'` here?

Reply:
(maybe a sniff for anything above 3 `"\n"` in a string? Unsure how complex that would be)

Reply:
Done.